### PR TITLE
Replace charts with Trading View

### PR DESCRIPTION
**Pull request by avclarke**
Removes the old charts and multiple chart options and replaces them with trading view candlestick chart only.

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 build
+public
```
Comment (JChiaramonte7) [inline .prettierignore:3]: Is there a reason why we're committing the public folder now? PR seems to add a lot of noise files.

Comment (avclarke) [reply .prettierignore:3]: That's where the trading view lib lives, but I can scope this to just trading view folder

Comment (JChiaramonte7) [reply .prettierignore]: I see, I guess I just assumed TV was a plug and play npm package.

Comment (avclarke) [reply .prettierignore]: I presume they keep it out of npm as it's kind of private, have to go through that process to get access

**Review by JChiaramonte7 — APPROVED** [review]

**Pull request merged**